src/views/shared/SecurityBeacon.js

```diff
@@ -120,7 +137,8 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         <div class="circle right"></div>\
       </div>\
     </div>\
-    <div class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+    <div aria-live="polite" role="alert" class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+      <span class="auth-beacon-description" style="height: 0; display: block; overflow: hidden"></span>\
```

Comment:
The idea here is to make it invisible to the user, but still "displayed" from the browser's perspective. Elements with `display: none` aren't read by screen readers (to the best of my knowledge).

That said, I could add a class with these attributes. Would courage be the right place for it?